### PR TITLE
better handing of list refreshing items

### PR DESCRIPTION
**Pull request by estebanlm**
it ensures the transmission happens

### CODE REVIEW

File: src/Spec-Core/SpecInterpreter.class.st [hunk 1]
```diff
@@ -11,6 +11,9 @@ Class {
 		'presenter',
 		'selector'
 	],
+	#classInstVars : [
```
Comment (jecisc) [inline src/Spec-Core/SpecInterpreter.class.st]: This variable should not be here.

File: src/Spec-Core/SpecPragmaCollector.class.st [hunk 2]
```diff
@@ -9,6 +9,9 @@ Class {
 	#instVars : [
 		'behavior'
 	],
+	#classInstVars : [
```
Comment (jecisc) [inline src/Spec-Core/SpecPragmaCollector.class.st]: This variable should not be here.

File: src/Spec-Core/ComposablePresenter.class.st [hunk 3]
```diff
@@ -1191,13 +1189,14 @@ ComposablePresenter >> show [
 { #category : #accessing }
 ComposablePresenter >> spec [
 
-	^ spec
+	^ adapter
 ]
 
 { #category : #accessing }
-ComposablePresenter >> spec: aSpec [
+ComposablePresenter >> spec: anAdapter [
```
Comment (jecisc) [inline src/Spec-Core/ComposablePresenter.class.st]: We should deprecate those methods with a rewrite rule but I can do it later.

File: src/Spec-Inspector/EyeInspector.class.st [hunk 4]
```diff
@@ -11,6 +11,9 @@ Class {
 		'list',
 		'description'
 	],
+	#classInstVars : [
+		'useAutoRefresh'
```
Comment (jecisc) [inline src/Spec-Inspector/EyeInspector.class.st]: This variable should not be here.

Comment (estebanlm) [reply src/Spec-Inspector/EyeInspector.class.st]: uh? 
I didn't add them.

Comment (jecisc) [reply src/Spec-Inspector/EyeInspector.class.st]: I removed them in a previous commit. Maybe Iceberg got mixup?

File: src/Spec-Core/SpecPragmaCollector.class.st (hunk 2 above)
Comment (jecisc) [reply src/Spec-Core/SpecPragmaCollector.class.st]: This one should still be removed.

File: src/Spec-Core/SpecInterpreter.class.st (hunk 1 above)
Comment (estebanlm) [reply src/Spec-Core/SpecInterpreter.class.st]: which one?

File: src/Spec-Core/SpecPragmaCollector.class.st (hunk 2 above)
Comment (estebanlm) [reply src/Spec-Core/SpecPragmaCollector.class.st]: still... I didn't touch any of this.
if they are there, is not my fault :)

**Pull request merged**